libraries/botbuilder-vendors/src/index.ts

```
@@ -0,0 +1,75 @@
+            hasLocalDependencies: true,
+            ignorePath: [`**/${VENDORS_DIR}/**/*`],
+        });
+
+        logger.summary({ action, vendors: allVendors.length, workspaces: workspaces.length });
+
+        for (const { pkg, absPath } of workspaces) {
+            logger.package.header({ name: pkg.name });
+
+            const pkgDir = path.dirname(absPath);
+            const directory = pkg.localDependencies!.__directory ?? 'vendors';
+            delete pkg.localDependencies!.__directory;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)